using-mobile-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -547,6 +547,32 @@ ACPCore.log(ACPMobileLogLevel.debug, tag: "MyClassName", message: "Provided data
 {% endtab %}
 {% endtabs %}
 
+### Handle open url action
```

Comment:
please change "url" to "URL" in the heading

Reply:
fixed.